app/api/logging/flask_logger.py

```diff
@@ -73,6 +74,11 @@ def add_extra_data_to_current_request_logs(
     setattr(flask.g, EXTRA_LOG_DATA_ATTR, extra_log_data)
 
 
+def _track_request_start_time() -> None:
+    """Store the request start time in flask.g"""
+    flask.g.request_start_time = datetime.now()
```

Comment:
Let's use python's [time.perf_counter](https://docs.python.org/3/library/time.html#time.perf_counter) method which is more intended to use for performance measurements

Reply:
Changed